popup.js

```diff
@@ -0,0 +1,5 @@
+let logo = document.getElementById("logo");
+
+logo.onclick = function (element) {
```

Comment:
Since you're including your script tag at the bottom of your html body, this is probably fine. But in general you should make sure the dom has loaded before you add an event listener to an element. That guarantees that your event gets added and you don't hit errors on load. Also not sure if you were just naming your function parameter element for descriptive purposes, but since the parameter is an event object, I think it's standard to name it e, or ev for event

e.g. you could do:
`window.addEventListener('load', function() {
    document.getElementById("logo").onclick = function(e) {
      console.log(e.target)
    }
  }